index.js

```diff
@@ -7,6 +7,10 @@ async function run() {
     const platform = getVirtualEnvironmentName()
     const [engine, version] = parseRubyEngineAndVersion(core.getInput('ruby-version'))
 
+    if (!platform.startsWith('windows') && version.match(/(mingw|mswin)$/)) {
+      throw new Error(`ruby-${version} is only available on windows`)
+    }
```

Comment:
This shouldn't be necessary, getAvailableVersions + validateRubyEngineAndVersion will already check that below

Reply:
That's fine.  I thought a specific message might be helpful, I'll remove.